### PR TITLE
Feature 26 - dynamic fetching of "applied", "seen" and "type of work" tags in vacancies history

### PR DESCRIPTION
**Pull request by ferchodeleon**
## Checklist

- [x] I have followed the [contributor guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md)
- [ ] Tests are included
- [ ] Documentation is changed or added
- [ ] Related issue has been created
- [x] Commits messages follows [commit guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md/#Commits)

### Type of PR

- [x] Feature
- [ ] Bugfix
- [ ] Hotfix
- [ ] Chore

### Merge strategy

- [ ] Squash (default)
- [x] Merge commit (branch merges, usually used for merges to master)
- [ ] Rebase (preserve individual commits without merge commit)

### Purpose

> Enlist the purposes or write an abastract about it, please be detailed, remember that not all the members of the team have the context of yout tasks
### Relevant tickets

> Paste here the link to the task ticket
resolves [26]

### Related PR's

> Paste here the links to related PR's
### Screenshots

> Add screenshots that could help others do understand even more the objective of your pr

### CODE REVIEW

File: src/api/axiosClient.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 
 const axiosClient = axios.create();
 
-axiosClient.defaults.baseURL = 'https://gethired-enterprise.herokuapp.com';
+axiosClient.defaults.baseURL = 'https://gethiredplatzi.herokuapp.com/api/v1';
```
Comment (LeoCode0) [inline src/api/axiosClient.js:5]: Could be a better option put the baseURL into the .env file

File: src/components/ListOfVacancies/index.jsx [hunk 2]
```diff
@@ -1,18 +1,20 @@
 import React, { useState, useEffect } from 'react';
 import Button from '@mui/material/Button';
+import { Divider } from '@mui/material';
```
Comment (LeoCode0) [inline src/components/ListOfVacancies/index.jsx:3]: You can import Divider and Button both together in the same line, like this
```jsx
import { Divider, Button } from '@mui/material';
```

File: src/api/axiosClient.js (hunk 1 above)
Comment (ferchodeleon) [reply src/api/axiosClient.js:5]: If it is correct 😀, leave it for now for testing, and when Zajith uploads the changes from his branch it is changed to .env so as not to leave the URIs in the code 😀

Comment (ZajithCorro) [reply src/api/axiosClient.js:5]: Yep, now it uses an enviroment variable

File: src/components/Vacancy/index.jsx [hunk 3]
```diff
@@ -54,10 +74,10 @@ export const Vacancy = ({ title, salary, modality, applies, seen, description, i
 
 Vacancy.propTypes = {
     title: PropTypes.string.isRequired,
+    checked: PropTypes.bool.isRequired,
     salary: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
     modality: PropTypes.oneOf(['remote', 'face-to-face']).isRequired,
```
Comment (odagora) [inline src/components/Vacancy/index.jsx:79]: The validation values are: `['Remote', 'Part-time', 'Full-time']`

**Review by ZajithCorro — APPROVED** [review]
Everything looks fine 😄

**Review by odagora — COMMENTED** [review]
Please merge the latest changes from develop and add a new commit to the PR. The routes are not updated and might cause conflicts

**Review by odagora — APPROVED** [review]

**Review by odagora — APPROVED** [review]

**Pull request merged**